vertical-pod-autoscaler/e2e/v1/common.go

```diff
@@ -165,6 +171,117 @@ func GetHamsterPods(f *framework.Framework) (*apiv1.PodList, error) {
 	return f.ClientSet.CoreV1().Pods(f.Namespace.Name).List(options)
 }
 
+// NewTestCronJob returns a CronJob which does one of several testing behaviors.
+func NewTestCronJob(name, schedule string, concurrencyPolicy batchv1beta1.ConcurrencyPolicy,
```

Comment:
I'd cut it down as much as possible, we don't need to make this very extensible, so feel free to hardcode some stuff in the CronJob object instead of passing it here.

Reply:
I've cut it down as much as I could, although there still might be some lines in the CronJobSpec that might be unnecessary - I'm not really sure which ones are in order to create a job successfully.